source/includes/starting-compass-paste-string.rst

```diff
@@ -23,11 +27,24 @@
       .. include:: /includes/fact-hide-connection-string-password.rst
             
       To learn how to format the connection string for a deployment that is not 
-      hosted on |service|, see :manual:`Connection String URI Format </reference/connection-string>`.
+      hosted on |service|, see :manual:`Connection String URI Format
+      </reference/connection-string>`.
+      
+   .. step:: (Optional). Name your connection.
+
+      Use the name field to enter a name for your connection. If you do
+      not specify a name, Compass uses the cluster's hostname as the
+      connection name. 
+
+   .. step:: (Optional). Choose a color for your connection.
+
+      Use the color drop-down menu to select a label color for your
+      connection. When you connect to a connection, the label color is
+      the background color of tabs that reference your connection.
 
    .. step:: Connect to your cluster.
 
-      Click :guilabel:`Connect` or :guilabel:`Save & Connect` to navigate to the 
+      Click :guilabel:`Save` or :guilabel:`Save & Connect` to navigate to the 
       |compass-short| :doc:`Home Page </instance>`. 
       
       The :guilabel:`Save & Connect` button prompts you to save your connection 
```

Comment:
I don't think it will prompt you to save as a favorite connection--that's an option on the modal that users have to select before connecting/saving. Do you really mean that the connection will automatically save?

Reply:
I added an optional step about favoriting: https://preview-mongodbltranmdb2.gatsbyjs.io/compass/DOCSP-38430/connect/#-optional-.-favorite-your-connection.

Reply:
cool this looks good! this thread is making me realize that we should probably adjust our "Favorite Connection" page: https://preview-mongodbltranmdb2.gatsbyjs.io/compass/DOCSP-38430/connect/favorite-connections/ 

The page as-is is a little confusing because now _all_ connections are saved by default. We should probably define somewhere that favorites now just means that they appear at the top of the list, and adjust this page to match the multiple connections flow. Could we do this before closing the project?

Reply:
Yes! I opened [DOCSP-42696](https://jira.mongodb.org/browse/DOCSP-42696) to address this and will work on it before this final PR merges so that the changes are rolled in